tools/clang/test/HLSLFileCheck/hlsl/types/matrix/matrix_subscript.hlsl

```diff
@@ -0,0 +1,64 @@
+// RUN: %dxc -DMIDX=1 -DVIDX=2 -T ps_6_0 %s | FileCheck %s
+// RUN: %dxc -DMIDX=i -DVIDX=2 -T ps_6_0 %s | FileCheck %s
+// RUN: %dxc -DMIDX=1 -DVIDX=j -T ps_6_0 %s | FileCheck %s
+// RUN: %dxc -DMIDX=i -DVIDX=j -T ps_6_0 %s | FileCheck %s
```

Comment:
Let's add extra run line(s) to try lib target (like lib_6_3), and put [shader("pixel")] in front of entry point.  Same with other tests as it makes sense.